app/presenters/node_presenter.rb

```diff
@@ -10,4 +10,8 @@ def initialize(node, flow_presenter, state = nil, _options = {}, _params = {})
   def node_name
     @node.name
   end
+
+  def node_path
+    node_name.to_s.dasherize
+  end
```

Comment:
Do we have unit tests for this?

Reply:
I'm not too happy with the name, but couldn't think of a better alternative. `node_path_segment` perhaps. Happy to consider something else.

Reply:
The method is not unit tested directly, but it is tested indirectly in many of the tests where the node in the path has changed from underscore to dashed form. So yes it is tested.

Reply:
I've added a test for the node presenter which also tests node name